frontend/src/DetailedStats/CrowdPleaser/CrowdPleaser.css

```diff
@@ -1,8 +1,40 @@
-.crowd-pleaser-container{
+<<<<<<< Updated upstream
```

Comment:
we need to get rid of these conflicts but its already merge I am fixing them in another PR